MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+The current Maintainers Group for the Kubevirt Project consists of:
+
+| Name | Employer | Responsibilities |
+| ----------- | ------- | -------------------------------- |
+| [Marc Sluiter](https://github.com/slintes) | Red Hat | |
+| [David Vossel](https://github.com/davidvossel) | Red Hat | |
+| [Vladik Romanovsky](https://github.com/vladikr) | Red Hat | |
+| [Roman Mohr](https://github.com/rmohr) | Red Hat | |
+| [Artyom Lukianov](https://github.com/cynepco3hahue) | Red Hat | |
+| [Sebastian Scheinkman](https://github.com/SchSeba) | Red Hat | |
+| [Marcin Franczyk](https://github.com/mfranczy) | Red Hat | |
+| [Fabian Deutsch](https://github.com/fabiand) | Red Hat | |
+| [Stu Gott](https://github.com/stu-gott) | Red Hat | |
```

Comment:
I want to propose @vasiliy-ul from Suse who drove cgroupv2 for kubevirt and was quite active on bug fixing and is maintaining kubevirt for integration in ranger.

Reply:
@davidvossel @fabiand does that work for you? Like for @mlsorensen I would want to wait for a sign of life before just blindly adding them.

Reply:
Sure, I would be glad to contribute to KubeVirt in the new role if that was acceptable and fine with everyone. Probably I am still not fully familiar with all the project specifics but I am ready to dedicate additional effort to fill the gaps.

Reply:
Thanks for showing the openess, Vasily. We'll support you and are looking forward to your thoughts and opinions.